src/utils/processUnsealedResult.ts

```diff
@@ -0,0 +1,13 @@
+import { EventResponse } from '@fingerprintjs/fingerprintjs-pro-server-api'
+import { plugins, PluginType } from './registerPlugin'
+
+export async function processUnsealedResult(data: EventResponse | null): Promise<void> {
+  if (!data || !data.products.identification?.data?.requestId) {
+    return
+  }
```

Comment:
```suggestion
  if (!data?.products?.identification?.data?.requestId) {
    return
  }
```

Let's simplify it a bit